pages/blog/[slug].tsx

```diff
@@ -26,16 +28,41 @@ export default function BlogPost({ frontMatter, source }: Props) {
 	}, []);
 
 	return (
-		<Flex justifyContent="center" p="6">
-			<AuthorInfo show={showAuthor} data={frontMatter.author} />
-			<Box ml={{ base: '0', lg: '9' }} overflowY="scroll">
-				<BlogHeader metadata={frontMatter} />
-				<Box as="article" id="content" maxW="1000px">
-					<MDXRemote {...source} components={components} />
+		<>
+			<NextSeo
+				title={`${frontMatter.title} - BLACC Blog Post`}
+				canonical={canoncialUrl}
+				openGraph={{
+					type: 'website',
+					url: `https://www.blacc.xyz/blog/${frontMatter.title}`,
```

Comment:
Replace `https://www.blacc.xyz/` with `canonicalUrl`. The title should also be formatted to [kebab case](https://betterprogramming.pub/string-case-styles-camel-pascal-snake-and-kebab-case-981407998841), not sure if it's doing that here.

Reply:
Replacing title with slug since slug is alerady in kebab-case. I'll still add a utility to kebab-case strings